src/scss/common/cards.scss

```diff
@@ -248,10 +248,10 @@
     max-width: 322px;
     @include respond-above(sm) {
       margin: 1.5rem;
+      flex: 0 1 calc(50% - 1rem);
```

Comment:
i'd be interested to get an explanation for what this is doing for the benefit of the card? :)

Reply:
Thanks for spotting! I didn’t intend to include those changes; they were remnants from debugging. 
I’ll remove them from the PR as the majority of the custom styling for news-card isn't need except for `margin` and `max-width` 🙂